totalRP3/Modules/Dashboard/DashboardChangelog.lua

```diff
@@ -206,67 +171,8 @@ Thanks to {col:00ff00}Raenore{/col} for the help on some of these features, and
 - Removed advanced comms settings.
 ]];
 
---- Returns the fully formatted localized text for this view.
-local function getLocalizedText()
-	return strhtml(CHANGELOG_TEXT);
-end
-
---- Toggles a setting and displays a UI toast.
----  @param setting The setting to be toggled.
-local function toggleSetting(setting)
-	if Configuration.getValue(setting) then
-		Configuration.setValue(setting, false);
-		UITooltip.toast(loc.OPTION_DISABLED_TOAST, 3);
-	else
-		Configuration.setValue(setting, true);
-		UITooltip.toast(loc.OPTION_ENABLED_TOAST, 3);
-	end
-end
-
---- Mapping of URL handlers to register and unregister with this view.
-local URL_HANDLERS = {
-	right_click_profile = GenerateClosure(toggleSetting, "CONFIG_RIGHT_CLICK_OPEN_PROFILE"),
-	companion_speech = GenerateClosure(toggleSetting, "chat_npcspeech_replacement"),
-	default_color_picker = GenerateClosure(toggleSetting, "default_color_picker"),
-	disable_chat_ooc = GenerateClosure(toggleSetting, "chat_disable_ooc"),
-	open_mature_filter_settings = function()
-		Navigation.menu.selectMenu("main_91_config_main_config_register");
-	end,
-};
-
---- Tab view class that displays our changelog, and points out the awesome
----  new features that we've spent far too much time working on.
-local WhatsNewTabView = Class("TRP3_DashboardWhatsNewTabView", Dashboard.TabView);
-Dashboard.WhatsNewTabView = WhatsNewTabView;
-
-function WhatsNewTabView.static.getTabTitle()
-	return loc.DB_NEW;
-end
-
-function WhatsNewTabView:initialize(dashboard)
-	self.class.super.initialize(self, dashboard);
-
-	self.body = getLocalizedText();
-end
-
-function WhatsNewTabView:Hide()
-	self.class.super.Hide(self);
-
-	-- Unregister the hyperlink handlers.
-	for url in pairs(URL_HANDLERS) do
-		self.dashboard:UnregisterHyperlink(url);
-	end
-end
-
-function WhatsNewTabView:Show()
-	self.class.super.Show(self);
-
-	self.dashboard:SetHTML(getLocalizedText());
-
-	-- Register all the URL strings that allow people to toggle settings
-	-- from the view directly. Whoever thought of this feature was a genius,
-	-- by the way.
-	for url, handler in pairs(URL_HANDLERS) do
-		self.dashboard:RegisterHyperlink(url, handler);
-	end
+function TRP3_DashboardUtil.GenerateChangelog()
+	local text = CHANGELOG_TEXT;
+	text = TRP3_DashboardUtil.GenerateListMarkup(text);
```

Comment:
Not that it matters but couldn't it have just been `local text = TRP3_DashboardUtil.GenerateListMarkup(CHANGELOG_TEXT);`

Reply:
Yeah, leftover from some extra stuff that the code was doing when I put in bulleted lists I think.